pyvo/registry/regtap.py

```diff
@@ -539,7 +539,11 @@ def region_of_regard(self):
         by which a positional query against this resource should be "blurred"
         in order to get an appropriate match.
         """
-        return float(self.get("region_of_regard", 0))
+        # we get NULLs as NaNs here
+        val = self["region_of_regard"]
+        if val != val:
```

Comment:
I'm not sure this is the best way to test for `np.nan`s, or what other values would you expect here?

(e.g. `np.isnan(val)` or `np.isfinite(val)` is a cleaner check for NaNs)